lua/diagnosticls-configs/linters/swiftlint.lua

```diff
@@ -0,0 +1,25 @@
+local fs = require 'diagnosticls-configs.fs'
+
+return {
+  sourceName = 'swiftlint',
+  command = fs.get_executable('swiftlint'),
+  debounce = 100,
+  args = { 
+    'lint', 
+    '--use-stdin',
+    '--quiet', 
+    '--reporter', 
+    'json',
+  },
+  parseJson = {
+    line = 'line',
+    column = 'character',
+    message = '${reason} [${rule_id}]',
```

Comment:
Hey thanks for the PR! Any way you could prepend the linter name to the message string? Just adding `[swiftlint] ${reason} [${rule_id}]` should be fine.

Diagnostis LS doesn't do this by default if if there are multiple linters it's hard to differentiate from which linter it was from.

Reply:
LGTM